BaseFilter/sections/specific.txt

```diff
@@ -17385,6 +17385,7 @@ fijivillage.com#?#.contentleft > p > small:contains(/^ADVERTISEMENT$/)
 biertijd.com,biertijd.xxx#?##wrapper > div#sidebarcontainer > h2:contains(/Advertenties|Sponsors|Aanrader/)
 mumbaimirror.indiatimes.com#?##content > div[class*=" "]:has(> div > div > a[onclick])
 mumbaimirror.indiatimes.com#?#.latestCmmtData > .mrtoday > div[class*=" "]:has(> div > div > a[onclick])
+circleofcricket.com##div.container-full.add:has(> a[href]:first-child)
```

Comment:
Advertising request can be blocked instead of an item selector
`||circleofcricket.com/assets/adv/$image`